libraries/AP_HAL_ChibiOS/hwdef/ATOMRCF405NAVI-Deluxe/README.md

```diff
@@ -0,0 +1,144 @@
+# AtomRC F405-NAVI- Deluxe
+
+![ATOMRCF405NAVI-Deluxe](atomrcf405-navi-deluxe.jpg)
+
+
+the above image and some content courtesy of `ATOMRC <http://atomrc.com/>`__
+
+.. note::
+
+	Due to flash memory limitations, this board does not include all ArduPilot features.
+        See :ref:`Firmware Limitations <common-limited_firmware>` for details.
+
+# Specifications
+
+-  **Processor**
+
+   -  STM32F405RGT6 ARM (168MHz)
+   -  AT7456E OSD
+   -  ESP32 Bluetooth RF module
+
+
+-  **Sensors**
+
+   -  ICM-42688 IMU (accel, gyro)
+   -  SPL-06 barometer
+   -  Voltage & 120A current sensor
+
+
+-  **Power**
+
+   -  6V ~ 30V DC input power
+   -  5.15/6/7.15V seelctable, 5A BEC for servos
+   -  9V/12V, 2A BEC for video
+
+
+-  **Interfaces**
+
+   -  6x UARTS
+   -  12x PWM outputs (PWM12 defaults to serial LED)
+   -  1x RC input with inverter for SBUS/PPM
+   -  I2C port for external compass and airspeed sensor
+   -  Type-C USB port
+   -  SD Card Slot
+   -  6 pin JST-GH for GPS/Compass
+   -  6 pin JST-GH for DJI air units
+   -  6 pin JST-GH for remote USB/Buzzer included with autopilot
+
+
+-  **Size and Dimensions**
+
+   - 50mm x 30mm x 12mm
+   - 21g
+
+# Where to Buy
+
+[ATOMRC](<https://atomrc.com)
+
+# Pinout
+
+
+tbd
```

Comment:
no pinout?

Reply:
I have asked for images...told them no wiki page until I get them...will add when I get them

Reply:
I could take photos myself...but I think we can wait....but we do have users that want to move from the shipped Inav to AP and I have sent several firmware out already...don't think we should wait to get it into "latest"